dataprocess/dataprocess.go

```diff
@@ -0,0 +1,38 @@
+// Package dataprocess manages code related to the data-accepting process
+package dataprocess
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"os"
+
+	pc "go.viam.com/rdk/pointcloud"
+)
+
+// WritePCDToFile Encodes the pointcloud and then saves it to the passed filename.
+func WritePCDToFile(ctx context.Context, pointcloud pc.PointCloud, filename string) error {
+	buf := new(bytes.Buffer)
+	err := pc.ToPCD(pointcloud, buf, 1)
+	if err != nil {
+		return err
+	}
+	return WriteBytesToFile(buf.Bytes(), filename)
+}
+
+// WriteBytesToFile Writes the passed bytes to the passed filename.
```

Comment:
[nit] Reflect the style in RDK

```suggestion
// WriteBytesToFile writes the passed bytes to the passed filename.
```